inlong-sort/sort-connectors/doris/src/main/java/org/apache/inlong/sort/doris/table/DorisDynamicTableSink.java

```diff
@@ -47,6 +55,9 @@ public class DorisDynamicTableSink implements DynamicTableSink {
     private final String inlongMetric;
     private final String auditHostAndPorts;
     private final Integer parallelism;
+    private static final String UNIQUE_KEYS_TYPE = "UNIQUE_KEYS";
+    public static final String COLUMNS_KEY = "columns";
+    public static final String DORIS_DELETE_SIGN = "__DORIS_DELETE_SIGN__";
```

Comment:
Do these two constants need to be set to `public`?

Reply:
done.